readthedocs/projects/views/private.py

```diff
@@ -1181,20 +1180,25 @@ def get_context_data(self, **kwargs):
 
         # Count of views for top pages over the month
         top_pages = PageView.top_viewed_pages(project, limit=25)
-        top_viewed_pages = list(zip(
-            top_pages['pages'],
-            top_pages['view_counts']
-        ))
+        broken_links = PageView.top_viewed_pages(
```

Comment:
we could use `top_200_pages` and `top_404_pages` to make them more clear. Also in the template context.